### PR TITLE
MM-50940 Add community data to looker

### PR DESCRIPTION
**Pull request by sri-byte**
Impact: Adds community data from RC and telemetry prod sources for Developer debugging. 
Ticket link https://mattermost.atlassian.net/browse/MM-50940

### CODE REVIEW

File: transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        'stg_mm_telemetry_prod__tracks' AS _source_relation,
+        {{dbt_utils.star(ref('stg_mm_telemetry_prod__tracks'))}}
+        , received_at::date as received_at_date
+        , timestamp::date as event_date
+     FROM
+       {{ ref('stg_mm_telemetry_prod__tracks') }} 
+    WHERE server_id = '93mykbogbjfrbbdqphx3zhze5c'
+{% if is_incremental() %}
+    AND received_at_date >= (SELECT MAX(received_at_date) FROM {{ this }}) 
+    AND _source_relation = 'stg_mm_telemetry_prod__tracks'
```
Comment (ifoukarakis) [inline transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql:19]: This expression must be inside the `SELECT MAX(...) ` sub-query. Please also check the comments in the previous PR: https://github.com/mattermost/mattermost-data-warehouse/pull/1192

**Review by ifoukarakis — CHANGES_REQUESTED** [review]
Please see comments in original PR: https://github.com/mattermost/mattermost-data-warehouse/pull/1192

Build is also failing.

File: transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql (hunk 1 above)
Comment (sri-byte) [reply transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql]: Addressed all the comments in the previous closed PR

Comment (sri-byte) [reply transform/mattermost-analytics/models/marts/web_app/community/community_tracks.sql]: Moved this expression inside the sub query

**Review by ifoukarakis — APPROVED** [review]

**Comment by ifoukarakis** [comment]
Can you also please add an exposure?

**Pull request merged**